### PR TITLE
BaSD-MS-34: Clients redux

### PR DESCRIPTION
**Pull request by Pablo-Balbo**
## Description:
Implementation of Redux to the project made with React.
The Create, Update and Delete functions of the Clients resource were adapted to Redux.
### Steps to test Client's functions:
1. Run `npm run start` on both front and back-end apps.
2. Search for **Clients** option on nav bar and click on it.
3. Click **add** or **edit** button to open form.
4. Complete all the fields with the corresponding data. 
If you are trying to update a client, its information will be loaded in the form automatically. Just change the necessary data.
If you are trying to add a new client, just complete with all the required information.
5. Click "Confirm" button.
6. Check if new client is shown in the Client's list or in case of having edited a client, check if the changed information is correct.
### Reviewers:
- @germanradium 
- @nicolasblasco 
- @GuidoGlielmi 
- @ValentinaCollareta

### CODE REVIEW

**Review by GuidoGlielmi — DISMISSED** [review]

File: src/Components/Clients/Form/index.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
```
Comment (nicolasblasco) [inline src/Components/Clients/Form/index.js:2]: @Pablo-Balbo we should remove Link and use useHistory instead

File: src/Components/Clients/Form/index.js [hunk 2]
```diff
@@ -251,7 +219,7 @@ function Form() {
           </Link>
```
Comment (nicolasblasco) [inline src/Components/Clients/Form/index.js:219]: @Pablo-Balbo we should remove Link and use useHistory instead

Comment (GuidoGlielmi) [inline src/Components/Clients/Form/index.js:219]: sacaleeeeeeeeeeeeeeeee el link

File: src/Components/Clients/Form/index.js (hunk 1 above)
Comment (Pablo-Balbo) [reply src/Components/Clients/Form/index.js]: Done.

File: src/Components/Clients/Form/index.js (hunk 2 above)
Comment (Pablo-Balbo) [reply src/Components/Clients/Form/index.js]: Done.

Comment (Pablo-Balbo) [reply src/Components/Clients/Form/index.js]: Dooooooooneeeeeeeee.

**Review by ValentinaCollareta — APPROVED** [review]

**Pull request merged**